bcda/main.go

```diff
@@ -386,14 +386,24 @@ func createAccessToken(acoID, userID string) (string, error) {
 		return "", errors.New(strings.Join(errMsgs, "\n"))
 	}
 
+	db := database.GetGORMDbConnection()
+	defer db.Close()
+	var aco models.ACO
+	var user models.User
+
+	if db.First(&aco, "UUID = ?", acoID).RecordNotFound() {
+		return "", fmt.Errorf("unable to locate ACO with id of %s", acoID)
+	}
+	if db.First(&user, "UUID = ?", userID).RecordNotFound() {
+		return "", fmt.Errorf("unable to locate User with id of %s", userID)
+	}
```

Comment:
Should we also check that the user is part of the ACO?

Reply:
agreed, but let's take this a step further and remove the need for an ACO entirely.  Users are scoped to a single ACO, just figure out the ACO from the user.  The command will then just take a user ID as the only input.  Making this change will just require deleting some existing code.